src/connector/src/sink/doris_starrocks_connector.rs

```diff
@@ -34,29 +36,30 @@ const BUFFER_SIZE: usize = 64 * 1024;
 const MIN_CHUNK_SIZE: usize = BUFFER_SIZE - 1024;
 const DORIS_SUCCESS_STATUS: [&str; 2] = ["Success", "Publish Timeout"];
 pub(crate) const DORIS_DELETE_SIGN: &str = "__DORIS_DELETE_SIGN__";
+pub(crate) const STARROCKS_DELETE_SIGN: &str = "__op";
+const STARROCK_MYSQL_PREFER_SOCKET: &str = "false";
+const STARROCK_MYSQL_MAX_ALLOWED_PACKET: usize = 1024;
+const STARROCK_MYSQL_WAIT_TIMEOUT: usize = 31536000;
```

Comment:
This is because initially we were directly querying in MySQL. Since Starrock doesn't have this table, it required us to set up bug prevention. We set the default value, but the value 31536000 was incorrect due to an error in the location of the query. After checking the official website, it should be 28800, and it has been updated.